python_modules/dagster/dagster/_cli/start.py

```diff
@@ -0,0 +1,124 @@
+import os
+import subprocess
+import sys
+import time
+import warnings
+
+import click
+
+from dagster._cli.job import apply_click_params
+from dagster._cli.workspace.cli_target import (
+    get_workspace_load_target,
+    python_file_option,
+    python_module_option,
+    workspace_option,
+)
+from dagster._core.instance import DagsterInstance
+from dagster._serdes import serialize_dagster_namedtuple
+
+
+def start_command_options(f):
+    return apply_click_params(
+        f,
+        workspace_option(),
+        python_file_option(),
+        python_module_option(),
+    )
+
+
+@click.command(
+    name="start",
+    help=(
+        "Start a local deployment of Dagster, including dagit running on localhost and the"
+        " dagster-daemon running in the background"
+    ),
+)
+@start_command_options
+@click.option(
+    "--grpc-log-level",
```

Comment:
Should this be user-code-server-log-level? We use grpc colloquially to refer to these, but it's just the transport protocol, right?

Reply:
we are inconsistent (the command is called dagster api grpc not dagster api code-server) but I agree with the feedback